doc/dune.inc

```diff
@@ -269,6 +269,15 @@
  (package dune)
  (files   dune-subst.1))
 
+(rule
+ (with-stdout-to dune-tools.1
+  (run dune tools --help=groff)))
+
+(install
+ (section man)
+ (package dune)
+ (files   dune-tools.1))
+
```

Comment:
```suggestion
```

This seems to be added by mistake.

Reply:
Indeed a mistake; thanks, I've removed it.